index.html

```diff
@@ -237,7 +237,7 @@ <h2>How to Read this Document</h2>
       <li>Software developers who want to implement processors and APIs for
         JSON-LD</li>
       <li>Software developers who want to generate or consume Linked Data,
-        an RDF graph, or an <a>RDF Dataset</a> in a JSON syntax</li>
+        an RDF <a>graph</a>, or an <a>Dataset</a> in a JSON syntax</li>
```

Comment:
"a Dataset"

Reply:
The official terminology is indeed "RDF Dataset" alongside with "RDF graph". The term "Dataset" may be considered as too ambiguous. I would prefer keep this as is.